cli/cmd/sync_v3.go

```diff
@@ -413,9 +413,6 @@ func syncConnectionV3(ctx context.Context, source v3source, destinations []v3des
 				},
 			}
 			if err := writeClientsByName[destinationName].Send(wr); err != nil {
-				// Close the transformations pipeline when the destination is closed.
-				pipeline.Close()
```

Comment:
We already close the pipeline in the calling function https://github.com/cloudquery/cloudquery/blob/50ebeb7b78779281a1e22c79d676e3a14a8f668a/cli/internal/transformerpipeline/pipeline.go#L108